### PR TITLE
Panels UI modifications

### PR DESCRIPTION
**Pull request by ps48**
### Description
Panels UI modifications and bug fixes

### Issues Resolved
1. Update Color theming for visualizations
2. Added Edit button to visualizations
3. Renamed Refresh Panel button
4. Changed Placement of Add Visualization Button

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/visualizations/charts/bar.tsx [hunk 1]
```diff
@@ -25,10 +30,10 @@ export const Bar = ({ visualizations, barConfig = {}, layoutConfig = {}, isUniCo
   // Individual bars have different colors when stackLength = 1 and chart is not unicolor
   // Else each stacked bar has its own color using colorway
```
Comment (joshuali925) [inline dashboards-observability/public/components/visualizations/charts/bar.tsx:31]: could you also update comment here

Comment (ps48) [reply dashboards-observability/public/components/visualizations/charts/bar.tsx:31]: Sure 👍🏽

**Comment by joshuali925** [comment]
your 48a4b5d reverted some committed change?

**Comment by ps48** [comment]
> your [48a4b5d](https://github.com/opensearch-project/trace-analytics/commit/48a4b5dbd555bbaea96adcf414f77729c834206e) reverted some committed change?

This is strange, I had to force amend my DCO commits, this change ruined the branch 😞 . I'll close this PR and create a new one with fresh branch. Modifying this branch again may create some issues in future.

**Pull request closed without merging**